schema/artist/index.js

```diff
@@ -426,7 +426,7 @@ const ArtistType = new GraphQLObjectType({
         resolve: ({ id }, options) => {
           return gravity('related/sales', defaults(options, {
             artist_id: id,
-            sort: '-end_at',
+            sort: 'timely_at,name',
```

Comment:
Wasn't sure about this... previously it was sorting by `-end_at`, which would grab sales that are ending farther in the future first. Force is passing `size: 2` and `live: true` here, which suggest that we should be grabbing the sales that are ending (or starting live) soonest to display. 

That being said, this endpoint also takes a `sort` param. Should force be specifying the sort here? Would that cause this `sort` to be overridden?